applications/SwimmingDEMApplication/custom_utilities/L2_error_projection_utility.h

```diff
@@ -47,14 +47,25 @@ L2ErrorProjection()
 
 virtual ~L2ErrorProjection(){}
```

Comment:
Here you are not really projecting, but merely calculating L2-norms. Please change name accordingly